src/devices/Uln2003/Uln2003.cs

```diff
@@ -126,7 +127,7 @@ public void Stop()
         public void Step(int steps)
         {
             double lastStepTime = 0;
-            _stopwatch = Stopwatch.StartNew();
+            _stopwatch.Start();
```

Comment:
`Restart()` will be equivalent to previous logic

Reply:
Fixed